lib/secure-boot-lib.robot

```diff
@@ -247,41 +247,3 @@ Go To Secure Boot Menu Entry
         ${index}=    Evaluate    ${index} - 1
     END
     Press Key N Times And Enter    ${index}+1    ${ARROW_DOWN}
-
-Select Attempt Secure Boot Option
```

Comment:
We work on making secure boot work properly in:
- https://github.com/Dasharo/open-source-firmware-validation/pull/117
- https://github.com/Dasharo/open-source-firmware-validation/pull/118

Let's do not introduce any SB changes in this MR. Let's assume the SB is disabled if needed (leave comment indicating that).